src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -182,6 +184,14 @@ private function getErrorMessages(Exception $e): array
                     'Admin.Notifications.Error'
                 ),
             ],
+            DuplicateFeatureValueAssociationException::class => $this->trans(
+                'You cannot associate the same feature value more than once.',
+                'Admin.Notifications.Error'
+            ),
+            InvalidAssociatedFeatureException::class => $this->trans(
+                'You cannot associate a feature value to another feature.',
```

Comment:
I'm not sure to understand what you mean here?

Reply:
I'm not fully satisfied either 😅 It's not supposed to happen but the idea is that you cannot associate a feature value that is associated to Feature A to another Feature B. For example in default data we have two features:

- Composition
  - Polyester
  - Wool
  - Ceramic
  - ...
- Property
  - Long sleeves
  - Short sleeves
  - ...

So you can't associate the value "Ceramic" to the feature "Property" It is not supposed to happen anyway since when you select a feature, the possible values are automatically updated with appropriate values. I guess it could happen if you have edited the value while you were selecting a feature, and you save after it's been modified.

Maybe we could rephrase it to something like `The property you selected does not belong to the selected feature.` ?

Reply:
Thank you it is way clearer now.
What do you think of: 'The selected value already belongs to another feature.'?

Reply:
Thank you, it is clearer now! :)
What do you think of: 
'The selected value already belongs to another feature.'
or
'A value cannot be associated with two different features.'

Reply:
MM, I think the first one is more suited but I would drop the already `The selected value belongs to another feature.` because it sort of implies the user was moving the value to another feature, or it didn't have a feature yet Whereas it is trying to assign it to a product 🤔

Reply:
Sounds good to me!